packages/styles/src/components/list-group.scss

```
@@ -1,162 +1,103 @@
+.list-group {
   background-color: transparent;
-  font-weight: normal;
-  text-decoration: none;
-  border-collapse: collapse;
-  border-right: 0;
-  border-left: 0;
-
-  @include media-breakpoint-up(rg) {
-    padding-left: map.get(spacing.$post-sizes, 'large');
-
-    &-minimal {
-      padding-left: 0;
-    }
-
-    &-transparent {
-      padding-right: map.get(spacing.$post-sizes, 'mini');
-      padding-left: map.get(spacing.$post-sizes, 'regular');
-    }
-  }
-
-  @include media-breakpoint-up(lg) {
-    &-big {
-      padding: map.get(spacing.$post-sizes, 'small-large') map.get(spacing.$post-sizes, 'mini')
-        map.get(spacing.$post-sizes, 'small-large') map.get(spacing.$post-sizes, 'regular');
-    }
-  }
+  list-style: none;
+  padding: 0;
+
+  > li {
+    border: unset;
+    padding: 0;
+    background: transparent;
+
+    .list-group-link,
+    .list-group-document,
+    .list-group-switch {
+      display: flex;
+      position: relative;
+      padding-block-start: tokens.get('list-item-padding-block-start');
+      padding-block-end: tokens.get('list-item-padding-block-end');
+      padding-inline-start: tokens.get('list-item-padding-inline-start');
+      padding-inline-end: tokens.get('list-item-padding-inline-end');
+      border-radius: tokens.get('list-item-focus-outline-radius'); // Used for the focus only
+      color: tokens.get('list-item-enabled-fg');
+
+      &::after {
+        display: block;
+        content: '';
+        position: absolute;
+        inset: auto 0 0;
+        border-block-end: tokens.get('list-item-border-width') tokens.get('list-item-border-style')
+          tokens.get('list-item-enabled-border');
+      }
 
-  &-svg {
-    flex: 0 0 auto;
-  }
+      &:hover {
+        color: tokens.get('list-item-hover-fg');
 
-  &-left-icon {
-    justify-content: flex-start;
-    padding-left: map.get(spacing.$post-sizes, 'mini');
+        &::before {
+          border-block-end-color: tokens.get('list-item-hover-border');
+        }
+      }
 
-    @include media-breakpoint-up(rg) {
-      padding-left: map.get(spacing.$post-sizes, 'small-regular');
+      @include utilities-mx.focus-style(
+        $offset: tokens.get('list-item-focus-outline-offset'),
+        $width: tokens.get('list-item-focus-outline-width'),
+        $color: tokens.get('list-item-focus-outline-color'),
+        $style: tokens.get('list-item-focus-outline-style')
+      );
     }
 
-    .list-group-item-svg {
-      margin-right: map.get(spacing.$post-sizes, 'mini');
+    .list-group-link,
+    .list-group-document {
+      align-items: center;
+      flex-grow: 1;
+      text-decoration: none;
+      gap: tokens.get('list-item-gap-inline');
 
-      @include media-breakpoint-up(rg) {
-        margin-right: map.get(spacing.$post-sizes, 'small-regular');
+      > *:last-child {
+        margin-left: auto;
       }
     }
-  }
 
-  &-no-border {
-    border: 0;
-  }
+    .list-group-switch {
+      justify-content: space-between;
```

We need to reset the `margin-bottom` here, which is defined on the `.form-check` class.

```suggestion
      justify-content: space-between;
      margin-bottom: 0;
```